example/Jamroot

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
I added this so that `b2` could run on the example.  Does `docca` assume a `Jamroot` already exists for the project?  If so, I will remove.